kahuna/public/js/edits/labeller.css

```diff
@@ -0,0 +1,75 @@
+/* ==========================================================================
+   Labeller
+   ========================================================================== */
+.labeller {
+    font-size: 1.3rem;
+    display: flex;
+}
+
+.label {
```

Comment:
This class seems to appear in other places outside of the labeller directive (e.g. in the gr-panel component). Let's try and be mindful of isolating classes that have side-effects elsewhere.